src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1254,10 +1265,20 @@
         <summary>List of parameters of the anonymous method expression, or null if there no parameters are specified.</summary>
       </PropertyComment>
     </Field>
-    <Field Name="Body" Type="CSharpSyntaxNode" Override="true">
-      <Kind Name="Block"/>
+    <Field Name="Block" Type="BlockSyntax" Override="true">
       <PropertyComment>
-        <summary>BlockSyntax node representing the body of the anonymous method.</summary>
+        <summary>
+          BlockSyntax node representing the body of the anonymous function.
+          This will never be null.
+        </summary>
+      </PropertyComment>
+    </Field>
+    <Field Name="ExpressionBody" Type="ExpressionSyntax" Optional="true" Override="true">
+      <PropertyComment>
+        <summary>
+          Inherited from AnonymousFunctionExpressionSyntax, but not used for 
+          AnonymousMethodExpressionSyntax.  Thsi will always be null.
```

Comment:
>Thsi [](start = 44, length = 4)

nit: spelling

Reply:
@RikkiGibson I have a couple of followup PRs.  Will fixup there!  That way i don't need to cause CI to rerun :D

Reply:
I've fixed this up in my other branch.  Thanks!